db/migrate/20240111104956_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```

Comment:
- [ ] **[OPTIONAL]** The `text` column should be of type `t.text` so that it can save more text, as `t.string` has lower storage capability.